### PR TITLE
[CPU] fix: supported Tile with more than 5 dims

### PR DESCRIPTION
**Pull request by antonvor**
Issue 35807

### CODE REVIEW

File: inference-engine/tests/functional/plugin/shared/include/single_layer_tests/tile.hpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+#include <tuple>
+#include <vector>
+#include <string>
+#include <memory>
+
+#include "functional_test_utils/layer_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+
+typedef std::vector<size_t> tileSpecificParams;
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/include/single_layer_tests/tile.hpp:15]: Minor comment, Type name should look as `TileSpecificParams`

File: inference-engine/tests/functional/plugin/shared/src/single_layer_tests/tile.cpp [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    tileSpecificParams tileParams;
+    std::vector<size_t> inputShape;
+    auto netPrecision   = InferenceEngine::Precision::UNSPECIFIED;
+    std::tie(tileParams, netPrecision, inputShape, targetDevice) = this->GetParam();
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+    auto paramOuts = ngraph::helpers::convert2OutputVector(
+            ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(params));
+    auto repeats = std::make_shared<ngraph::opset1::Constant>(ngraph::element::i64, std::vector<size_t>{tileParams.size()}, tileParams);
+    auto tile = std::make_shared<ngraph::opset1::Tile>(paramOuts[0], repeats);
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/shared/src/single_layer_tests/tile.cpp:41]: Looks like, we can move tile logic creation to the builder

File: inference-engine/tests/functional/plugin/shared/include/single_layer_tests/tile.hpp (hunk 1 above)
Comment (antonvor) [reply inference-engine/tests/functional/plugin/shared/include/single_layer_tests/tile.hpp]: thanks, changed

File: inference-engine/tests/functional/plugin/shared/src/single_layer_tests/tile.cpp (hunk 2 above)
Comment (antonvor) [reply inference-engine/tests/functional/plugin/shared/src/single_layer_tests/tile.cpp]: Changed

**Review by iefode — APPROVED** [review]
Test part LGTM

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**